File: samples/CS/02-ApplicationMenuButton/RibbonItems.cs
using System;
using System.Windows.Forms;

namespace WinForms.Ribbon
{
    partial class RibbonItems
    {
        public void Init()
        {
            ApplicationMenu.TooltipTitle = "Menu";
            ApplicationMenu.TooltipDescription = "Application main menu";

            ButtonNew.Click += new EventHandler<EventArgs>(_buttonNew_Click);
        }

        void _buttonNew_Click(object sender, EventArgs e)
        {
            MessageBox.Show("New button pressed");
        }

        public void Load()
        {
        }

    }
}

File: samples/VB/02-ApplicationMenuButton/RibbonItems.vb
Imports System

Namespace WinForms.Ribbon

    Partial Class RibbonItems

        Public Sub Init()
			_ApplicationMenu.TooltipTitle = "Menu"
			_ApplicationMenu.TooltipDescription = "Application main menu"

			AddHandler _ButtonNew.Click, AddressOf _buttonNew_ExecuteEvent
		End Sub

		Private Sub _buttonNew_ExecuteEvent(ByVal sender As Object, ByVal e As EventArgs)
			MessageBox.Show("new button pressed")
		End Sub

	End Class
End Namespace
